src/components/Sidebar/NavContent.jsx

```diff
@@ -118,9 +121,9 @@ function NavContent({ handleDrawerToggle }) {
                   <Link
                     component={NavLink}
                     id="bond-nav"
-                    to="/bonds"
+                    to="/bonds-v2"
                     isActive={(match, location) => {
-                      return checkPage(match, location, "bonds");
+                      return checkPage(match, location, "bonds-v2");
```

Comment:
does bonds-v2 page have the ability to claim bonds from v1?

Reply:
nope, but there is a link to v2